apps/server/src/services/project-service/ProjectService.ts

```diff
@@ -97,8 +95,8 @@ async function loadProject(projectData: DatabaseModel, projectName: string) {
   await initRundown(firstRundown, projectData.customFields);
 
   // persist the project selection
-  const newName = getFileNameFromPath(pathToNewFile);
-  await setLastLoadedProject(newName);
+  const newName = getFileNameFromPath(fileName);
```

Comment:
question: `getFileNameFromPath` retrieves the name from a path. However, we are already passing it a filename
This brings me to a few doubts:
- perhaps we should not be using this function
- `newName` provides incorrect expectations since there are no changes to the name

Reply:
I am also a bit unsure of the `projectData` since it is not here we are parsing it and it seams to only be used as a fallback value in `initPersistence` ?

Reply:
it looks like we use the projectData as data source but I agree that it seems unnecessary and the filename should be enough. I suggest we move on so we dont risk breaking things unless we have a clear idea on how to re-write it